Index: .idea/dataSources/744617a1-0b5c-40b2-b37f-b6b840bea59f/storage_v2/_src_/schema/python_database.fkkjPA.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/dataSources/744617a1-0b5c-40b2-b37f-b6b840bea59f/storage_v2/_src_/schema/python_database.fkkjPA.meta b/.idea/dataSources/744617a1-0b5c-40b2-b37f-b6b840bea59f/storage_v2/_src_/schema/python_database.fkkjPA.meta
new file mode 100644
--- /dev/null	(date 1718613477909)
+++ b/.idea/dataSources/744617a1-0b5c-40b2-b37f-b6b840bea59f/storage_v2/_src_/schema/python_database.fkkjPA.meta	(date 1718613477909)
@@ -0,0 +1,2 @@
+#n:python_database
+!<md> [0, 0, null, null, -2147483648, -2147483648]
Index: .github/workflows/maven-publish.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.github/workflows/maven-publish.yml b/.github/workflows/maven-publish.yml
new file mode 100644
--- /dev/null	(date 1718852639244)
+++ b/.github/workflows/maven-publish.yml	(date 1718852639244)
@@ -0,0 +1,18 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+# For more information see: https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#apache-maven-with-a-settings-path
+
+on:
+  push: # 在我推送代码的时候开始运行
+    branches:  
+    - main   
+# 工作
+jobs:
+  #工作1
+  job1:
+    # 运行在什么系统上
+    runs-on: ubuntu-latest
+    #执行命令
+    steps:
+      - run: ifconfig 
+      
+
